crates/wash/src/cli/dev.rs

```diff
@@ -322,29 +333,44 @@ impl CliCommand for DevCommand {
         info!("development session started successfully");
         info!(address = self.address, "listening for HTTP requests");
 
+        // Create a single spinner for the entire dev session
+        let dev_spinner = ProgressBar::new_spinner();
+        dev_spinner.set_style(
+            ProgressStyle::default_spinner()
+                .template("{spinner:.green} {msg}")
+                .unwrap(),
```

Comment:
```suggestion
                .context("failed to create spinner")?,
```

Would prefer to safely bail in the case that this fails for some reason